src/main.cpp

```diff
@@ -71,11 +70,14 @@ int main(int argc, char *argv[])
     QString error;
     QLockFile *lock = 0;
     if (!initSettings(settings.data(), &lock, error)) {
+        initTranslation();
```

Comment:
is there a reason to not just call initTranslation unconditionally before initSettings, instead of two seperate calls?